cli/src/commands/util.rs

```diff
@@ -108,16 +118,26 @@ fn cmd_util_completion(
 fn cmd_util_gc(
     ui: &mut Ui,
     command: &CommandHelper,
-    _args: &UtilGcArgs,
+    args: &UtilGcArgs,
 ) -> Result<(), CommandError> {
     if command.global_args().at_operation != "@" {
         return Err(user_error(
             "Cannot garbage collect from a non-head operation",
         ));
     }
+    match &args.prune {
+        None => {}
+        Some(s) if s.eq("now") => {}
+        _ => return Err(user_error("--prune only accepts 'now'")),
+    }
     let workspace_command = command.workspace_helper(ui)?;
-    // TODO: add command argument to specify the expiration time?
-    let keep_newer = SystemTime::now() - Duration::from_secs(14 * 86400);
+    let keep_newer = SystemTime::now()
+        - if args.prune.is_some() {
+            Duration::ZERO
+        } else {
+            Duration::from_secs(14 * 86400)
+        };
```

Comment:
better to map to duration by the previous `match` instead of reinterpreting `args.prune`

Reply:
Done. (Moved the assignment of the variable to the top.)